src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/Fragment/config.yml

```diff
@@ -4,6 +4,8 @@ imports:
 
 framework:
     http_method_override: false
+    esi: true
```

Comment:
I've reused this functional test, because I haven't found one that covers `esi` and `ssi`. However, I'd be happy to add a `FragmentEsi` and `FragmentSsi` test if that's preferred.